src/server/service/kv.rs

```diff
@@ -1320,6 +1321,12 @@ fn future_get<E: Engine, L: LockManager, F: KvFormat>(
     storage: &Storage<E, L, F>,
     mut req: GetRequest,
 ) -> impl Future<Output = ServerResult<GetResponse>> {
+    let tracker = GLOBAL_TRACKERS.insert(Tracker::new(RequestInfo::new(
+        req.get_context(),
+        RequestType::KvGet,
+        req.get_version(),
+    )));
+    set_tls_tracker(tracker);
```

Comment:
Where is the usage of this thread-local variable?

Reply:
I don't want to pass the token all over through function parameters. So, when the request just comes in, set it in the TLS storage, we can get the token any time later.

When we spawn a future from a context with a tls tracker, the `TrackedFuture` will maintain this tracker in the spawned futures.